centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -90,8 +96,10 @@ object Test {
   *
   * All operations are expected to return a Test type and implement the run method. These can then
   * be composed together via a for comprehension as a test formula and then run by some other entity.
+  *
+  * Stop! Do NOT add StrictLogging to this object. It will mess with the conformance testing.
```

Comment:
but...but... it was like that?

Reply:
ah, yeah, I should undo this now. It was unrelated to the actual problem I was having.